src/views/AwsProvisioner/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { hot } from 'react-hot-loader';
+import { Component } from 'react';
+import { Switch } from 'react-router-dom';
+import RouteWithProps from '../../components/RouteWithProps';
+import loadable from '../../utils/loadable';
+
+const ViewAwsHealth = loadable(() =>
+  import(/* webpackChunkName: 'Clients.ViewClients' */ './ViewAwsHealth')
+);
+
+@hot(module)
+export default class Clients extends Component {
+  render() {
+    const {
+      match: { path },
+      ...props
+    } = this.props;
+
+    return (
+      <Switch>
+        <RouteWithProps
+          path={`${path}/aws-health`}
+          {...props}
```

Comment:
What props need to be spread from view to view, and only a single view?

Reply:
I'm not sure I understand what you mean, but if you're asking whether `/aws-health` will need any props from this file, then probably not. I will remove it.

Reply:
Nevermind, `onSignIn`, `onSignOut`, and `user`are needed in the component.

Reply:
Ah I see. I'll have that fixed in my auth PR.